scripts/Dockerfile-build

```
@@ -15,5 +15,15 @@ ENV NAME=${NAME}
 
 RUN CGO_ENABLED=1 GOOS=linux go build -a -installsuffix cgo -ldflags "-extldflags \"-static\" -s -w -X main.GitCommit=$GIT_COMMIT -X main.BuildVersion=$VERSION -X main.BuildType=$NAME" -o owncast .
 
+# Create the image by copying the result of the build into a new alpine image
+FROM alpine
+RUN apk update && apk add ca-certificates && update-ca-certificates && apk add --no-cache ffmpeg ffmpeg-libs
```

```suggestion
RUN apk update && apk add --no-cache ffmpeg ffmpeg-libs ca-certificates && update-ca-certificates
```
Simplifies a bit more.  From my experience I haven’t needed `update-ca-certificates` but I don’t think it hurts any

---

Thanks!